src/containers/AdminAssignmentControl/components/AssignmentRow.tsx

```diff
@@ -37,17 +41,18 @@ const AssignmentRow: React.FC<Props> = (props) => {
   ) => onChange({ assignmentType });
 
   const handleChangeMaxCount = (
-    _event: React.FormEvent<unknown>,
-    maxRequestCount: string
-  ) =>
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const maxRequestCount = event?.target?.value;
     onChange({
       maxRequestCount: maxRequestCount ? parseInt(maxRequestCount, 10) : null
     });
+  };
 
   const handleChangeEscalationTags = (
     _event: React.ChangeEvent<any>,
-    value: Array<TagWithTitle> | null
-  ) => {
+    value: TagWithTitle[]
```

Comment:
nit: I prefer the `Array<type>` over `Type[]` style, reads better. Don't have to make that change, but we should settle on a convention 😓

Reply:
i like `Type[]` lol we should vote

Reply:
yeah fine with either but I also like Type[] lol